src/controllers/busController.js

```diff
@@ -1,11 +1,11 @@
 import { paginate } from 'paginate-info';
 import model from '../database/models';
 
-const { Bus } = model;
+const { Buses } = model;
 
 const createBus = async (req, res) => {
```

Comment:
Better use class as used in userController to login so that end-points can be accessed